tests/unit/resources/test_fcsfile.py

```diff
@@ -79,6 +81,31 @@ def test_gets_file_internal_compensation(ENDPOINT_BASE, client, fcs_files, spill
     assert type(comp) == Compensation
 
 
+@responses.activate
+def test_throws_correct_error_when_no_file_internal_compensation(
+    ENDPOINT_BASE, client, fcs_files
+):
+    # Given: An FcsFile with no spill string
+    file_data = fcs_files[0]
+    file_data["spillString"] = ""
```

Comment:
Our API returns `null`, not `""` in this case.

Reply:
Changed for the test; I needed to set `FcsFile._spill_string` to `""` instead of `None` because of `.get_file_internal_compensation`, but I've made this a bit simpler in the code now

Reply:
Actually, I've decided to stick with `return self._spill_string or ""`, because then `mypy` is happy with the consistent return type. This does _not_ cause a problem when PATCH-ing the file to the API

Reply:
The spill string has a meaning defined by the FCS specification, and the spec doesn't allow it to be blank (must be omitted instead). Can the types be altered to allow `None` instead?